prov/rxm/src/rxm_init.c

```diff
@@ -91,11 +91,11 @@ int rxm_info_to_core(uint32_t version, const struct fi_info *hints,
 	core_info->mode |= FI_RX_CQ_DATA | FI_CONTEXT;
 
 	if (hints) {
-		if (hints->caps & FI_TAGGED)
+		if (hints->caps & (FI_MSG | FI_ATOMIC | FI_TAGGED))
```

Comment:
Doesn't core_info->caps already have FI_MSG set if the user needs FI_MSG?  Likewise for FI_RMA below.

Reply:
You are right, thanks. I'll update to remove FI_MSG and FI_RMA from the respective tests.

Reply:
Updated, thanks.